### DISCUSSION TITLE
QR code help Dashcam

### DISCUSSION THREAD

**Discussion by Neacsucrs** [gopro/labs#201] [General]
Hello,

I am trying to set a qr code for go pro 11 with usb triger to use the camera as a dashcam. The problem is I cannot set the looping mode. Can someone help me with a qr code for this?

This is what I am using at this time.
mVr1080p60fVte0aWcNb0sMg1dLoV4S1!uN!S!u10E!R

Thank you

**Comment by dnewman-gpsw** [comment]
Enable the Looping mode preset on the camera first (as there is no QR code for that.)  

this is all you need: !uNmVL!S!u10E!R

General note: it is unnecessary to set all you parameters for every start-up, as these are auto saved in the preset.

**Comment by Neacsucrs** [comment]
It's perfect!Thank you ! 🙏🙏

On Fri, 20 Jan 2023 at 01:13, David Newman ***@***.***> wrote:

> Enable the Looping mode preset on the camera first (as there is no QR code
> for that.)
>
> this is all you need: !uNmVL!S!u10E!R
>
> General note: it is unnecessary to set all you parameters for every
> start-up, as these are auto saved in the preset.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gopro/labs/discussions/201#discussioncomment-4732609>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5LLUEIQKZJUSF73N2ZAGSDWTHDB7ANCNFSM6AAAAAAUA22I4Q>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Neacsu Cristian

Director General

S.C. Anasofia Events S.R.L.
0766216577
***@***.***
www.anastasiaevents.ro
Ciochina, Nr.16, Sector 4, Bucuresti
[image: facebook] <https://www.facebook.com/Anastasiaevents.ro/>
[image: instagram] <https://www.instagram.com/anastasiaevents.ro/>

**Comment by EpanchinIlya** [comment]
Hello!
I have firmware 02.10.70, gopro 11 black.
I Can't enable feature usb triger.
I use qrcode: !uN!S!u10E!
What am I doing wrong? Help me somebody!
The camera accepts the code and turns off, but when the usb is connected, only the charge LED lights up.
The camera itself does not turn on.

**Reply by dnewman-gpsw** [reply]
The old style USB trigger is almost obsolete as there are better ways now on HERO11 -- those new way sill need more documentation.  Seems that a bug, it making it more obsolete than I thought. However, if you scan this once !MWAKE=2 the odd style will work again.

**Comment by EpanchinIlya** [comment]
Thank you very much for your answer, everything worked out!
Now another problem. I want to use the camera as a dashcam too, but the camera is not powered by external usb during shooting. This can be seen in the decreasing battery level during shooting. How to make sure that the battery does not run out while shooting?

**Reply by dnewman-gpsw** [reply]
If you are not USB powered, you have to use the battery. Have a spare battery.

**Comment by EpanchinIlya** [comment]
I want the battery not discharge while shooting, but during the shooting, the usb charge stops, and the camera is powered by the battery, even if the usb power is connected. What can I do to powered from usb during filming? Thanks your help :)

**Reply by dnewman-gpsw** [reply]
The vamera will powered by USB when contented to USB.  It does not charge while capturing, but it does run the power from the external source.

**Reply by EpanchinIlya** [reply]
Unfortunately, my gopro 11 black camera is not powered by an external usb source while shooting. I checked in practice: in 25 minutes of shooting, the battery was discharged from 65% to 25% with usb connected (the camera is new).

**Reply by dnewman-gpsw** [reply]
Sounds like you might have an insufficient USB power source.  That is not my experience.  External USB power should be a minimum 2A, and report that to the camera upon connection.

**Reply by EpanchinIlya** [reply]
I connected to different sources of 5 volts. The experiment described was carried out on a laboratory power supply providing 5 volts, 3 amperes. So the problem is not the power supply.

**Reply by dnewman-gpsw** [reply]
It can be, as the GoPro is expecting the power supply to declare its rating, and I'm pretty sure lab power would not.  Try adding !MTUSB=1 this will bypass the power support current testing.

**Reply by EpanchinIlya** [reply]
Everything worked out, thank you very much for your help
